rca/api_content/content.py

```diff
@@ -0,0 +1,148 @@
+import logging
+from datetime import datetime
+
+import requests
+from django.http.request import QueryDict
+
+"""
+Static methods for adding content from the live RCA api
+"""
+
+logger = logging.getLogger(__name__)
+
+
+class CantPullFromRcaApi(Exception):
+    pass
+
+
+def parse_items_to_list(data, type):
+    items = []
+
+    for item in data["items"]:
+        _item = {}
+        if type == "News":
+            detail = item["meta"]["detail_url"] + "?fields=_,date,feed_image"
+        if type == "Event":
+            detail = item["meta"]["detail_url"] + "?fields=_,dates_times,feed_image"
+        if type == "Alumni stories":
+            detail = item["meta"]["detail_url"] + "?fields=_,feed_image,intro"
+
+        try:
+            response = requests.get(url=detail)
+            response.raise_for_status()
+        except requests.exceptions.HTTPError:
+            logger.exception("Error occured when fetching further detail data")
+
+        data = response.json()
```

Comment:
You can't do this. What if exception was raised? Do you still care about the JSON content? You should raise the `CantPullFromRcaApiError` in the `except` block probably.

Reply:
If, for whatever reason, you want the loop to continue regardless, you can also just `continue` within the except block, right?